--- a/configure.in
+++ b/configure.in
@@ -48,10 +48,14 @@
 ldflags_save="$LDFLAGS"
 if test -z "$GCC"; then
         case $host in 
-        arm-*-*)
+        arm-*-*) 
                 DEBUG="-g -D_ARM_ASSEM_" 
                 CFLAGS="-O -D_ARM_ASSEM_"
                 PROFILE="-p -g -O -D_ARM_ASSEM_" ;;
+        aarch64-*-*) 
+                DEBUG="-g"
+                CFLAGS="-O"
+                PROFILE="-g -p" ;;        
         *)
                 DEBUG="-g"
                 CFLAGS="-O"
@@ -64,7 +68,10 @@
                 DEBUG="-g -Wall -D__NO_MATH_INLINES -fsigned-char -D_ARM_ASSEM_"
                 CFLAGS="-O2 -D_ARM_ASSEM_ -fsigned-char"
                 PROFILE="-W -pg -g -O2 -D_ARM_ASSEM_ -fsigned-char -fno-inline-functions";;
-
+        aarch64-*-*)
+                DEBUG="-g -Wall -D__NO_MATH_INLINES -fsigned-char -DBYTE_ORDER=1234 -DLITTLE_ENDIAN=1234"
+                CFLAGS="-O2 -DBYTE_ORDER=1234 -DLITTLE_ENDIAN=1234 -fsigned-char"
+                PROFILE="-W -pg -g -O2 -DBYTE_ORDER=1234 -DLITTLE_ENDIAN=1234 -fsigned-char -fno-inline-functions";;
         *)
                 DEBUG="-g -Wall -D__NO_MATH_INLINES -fsigned-char"
                 CFLAGS="-O2 -Wall -fsigned-char"

