--- a/builds/unix/configure
+++ b/builds/unix/configure
@@ -2574,6 +2574,19 @@
 # Remember, the first character of IFS is used to create $*,
 # except with old shells:
 build_os=$*
+
+if test -n "$build_vendor"; then
+  if [[ $build_vendor == "pc" ]]; then
+#	echo "Removing pc from native compiler prefix!"
+#	ac_cv_build="$build_cpu-$build_os"
+	if test -n "$build_alias"; then
+		echo "pc detecred in prefix:: Overriding..."
+        echo "Using --build parameter as native compiler prefix!"
+		ac_cv_build="$build_alias"
+    fi
+  fi
+fi
+
 IFS=$ac_save_IFS
 case $build_os in *\ *) build_os=`echo "$build_os" | sed 's/ /-/g'`;; esac
 
@@ -12161,6 +12174,12 @@
 $as_echo "no" >&6; }
 fi
 
+echo "Setting proper assembler path, because otherwise it gets confused and uses the cross-compiler toolchain's"
+if test -n "${CC_BUILD}"; then
+  CC_BUILD_PATH=`which ${CC_BUILD}`
+  CC_BUILD_DIR=${CC_BUILD_PATH%/*}
+  CC_BUILD="${CC_BUILD} -B$CC_BUILD_DIR"
+fi
 
   test -z "${CC_BUILD}" && as_fn_error $? "cannot find native C compiler" "$LINENO" 5
 
--- a/include/freetype/config/ftoption.h
+++ b/include/freetype/config/ftoption.h
@@ -124,7 +124,7 @@
    * When this macro is not defined, FreeType offers alternative LCD
    * rendering technology that produces excellent output.
    */
-/* #define FT_CONFIG_OPTION_SUBPIXEL_RENDERING */
+#define FT_CONFIG_OPTION_SUBPIXEL_RENDERING
 
 
   /**************************************************************************
@@ -221,7 +221,7 @@
    *   options set by those programs have precedence, overwriting the value
    *   here with the configured one.
    */
-/* #define FT_CONFIG_OPTION_SYSTEM_ZLIB */
+#define FT_CONFIG_OPTION_SYSTEM_ZLIB
 
 
   /**************************************************************************
@@ -270,7 +270,7 @@
    *   options set by those programs have precedence, overwriting the value
    *   here with the configured one.
    */
-/* #define FT_CONFIG_OPTION_USE_PNG */
+#define FT_CONFIG_OPTION_USE_PNG
 
 
   /**************************************************************************

